src/general/OptionsMenu.tscn

```diff
@@ -795,7 +796,6 @@ margin_bottom = 32.0
 text = "RESET_INPUTS"
 
 [node name="Misc" type="Control" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer"]
-visible = false
```

Comment:
Related to the other `visible` change.

Reply:
That's just appeared when I make invisible misc back